portality/view/doajservices.py

```diff
@@ -16,6 +16,11 @@
 @ssl_required
 @write_required()
 def unlock(object_type, object_id):
+    # change from suggestion to application after redesign, that needs refactoring to make the code consistent
```

Comment:
I'm not sure I understand your comment here - you're changing the type to the one we don't use in order to avoid changing the old code?